src/Appwrite/GraphQL/Schema.php

```diff
@@ -234,23 +245,23 @@ protected static function collections(
                             $attributes
                         )
                     ),
-                    'resolve' => Resolvers::documentUpdate(
-                        $utopia,
+                    /*'resolve' => Resolvers::documentUpdate(
+                        $http,
                         $databaseId,
                         $collectionId,
                         $urls['update'],
                         $params['update'],
-                    )
+                    )*/
                 ];
                 $mutationFields[$collectionId . 'Delete'] = [
                     'type' => Mapper::model('none'),
                     'args' => Mapper::args('id'),
-                    'resolve' => Resolvers::documentDelete(
-                        $utopia,
+                    /*'resolve' => Resolvers::documentDelete(
```

Comment:
no resolve function?

Reply:
The [collection ](https://github.com/appwrite/appwrite/blob/5d70884e6cf4aa65e4da38f1bff870d4725964f0/src/Appwrite/GraphQL/Schema.php#L54) part was commented before, that's the only way to access that functions so they not in use right now, as of that they got commented out